dev-client/src/util.ts

```diff
@@ -138,3 +143,7 @@ export const isSiteManager = matchesRole([
   {kind: 'site', role: 'OWNER'},
   {kind: 'project', role: 'MANAGER'},
 ]);
+
+export const getSoilWebUrl = (coords: Coords) => {
```

Comment:
I feel that this file should be split apart in the future - any big file called `util` which mixes business and standard library-like concerns is a red flag to me

Reply:
comment: this url is actually getting passed along in the backend query so we don't need to construct it manually in the client

Reply:
Which property in the query response contains this url?

Reply:
Since you're now out until Tuesday - I'll resolve this conversation and we can coordinate on this when we get to https://github.com/techmatters/terraso-product/issues/653